components/Sidebar.tsx

```diff
@@ -13,8 +13,8 @@ export const Sidebar = (props: { isOpen: boolean; setIsOpen: any }) => {
 
   const menus = [
     {
-      href: '/admin/saml/config',
-      text: 'SAML Connections',
+      href: '/admin/connection',
+      text: 'Connections',
```

Comment:
Can we somehow mention Jackson for this menu as Directory Sync and Audit Logs will also come in the sidebar and the connection will be confusing?

Reply:
A better name could be "SSO Connections"?

Reply:
I vote for "SSO Connections"

Reply:
Noted.